File: tracker.c
# include <stdio.h>
# include <stdbool.h>
# define TOTAL 2440
# define RATE 1.2
# define INITIAL 7
# define FEE 9972
int main(void)
{
    int days = 0;
    while(true)
    {
        float infected = INITIAL;
        int total_infected = 0;
        int withdraw = (int)infected;
        float infected_Today = infected;

        printf("Enter an integer number of days to continue\nEnter a negative integer number to exit\n");
        scanf("%d", &days);
        if(days < 0)
        {
            puts("This is the end");
            break;
        }
        if(days == 0)
        {
            printf("Total number of infected are: %d\n", (int)infected);
            printf("The percentage of people infected: %f\n", (infected * 100/TOTAL));
            printf("The number of students withdrawing are: %d\n", withdraw);
            printf("Loss for institution is: %d\n", withdraw * FEE);
            continue;
        }
        for(int i = 0; i < days; i++)
        {
            infected +=  infected_Today * RATE;
            infected_Today *= 1.2; 
            if(i <= 14)
            {
                withdraw = (int)infected;
            }
        }
        if(infected > TOTAL)
        {
            puts("complete infection");
            printf("The whole loss is: %d\n", TOTAL * FEE);
            continue;
        }
        else
        {
            printf("Total number of infected are: %d\n", (int)infected);
            printf("The percentage of people infected: %f\n", (infected * 100/TOTAL));
            printf("The number of students withdrawing are: %d\n", withdraw);
            printf("Loss for institution is: %d\n", withdraw * FEE);
        }
    }
}

File: tracker.py

days = 0
total = 2440
fee = 9972
print("INSTRUCTIONS")
print("Floats would be trucated to int")
print("Enter a negative number of days to exit")
while(True):
    infected = 7
    withdraw = int(infected)
    infected_Today = infected
    days = int(input("Enter the number of days: "))
    if(days < 0):
        print("See you later")
        break
    if(days == 0):
        print("Total number of infected are: ", withdraw)
        print("The percentage of people infected: ", (infected * 100/total))
        print("The number of students withdrawing are: ", withdraw)
        print("Loss for institution is: ", withdraw * fee)
        continue
    for i in range (0, days):
        infected +=  infected_Today * 1.2
        infected_Today *= 1.2
        if(i <= 14):
            withdraw = int(infected)
    loss = withdraw * fee
    if(infected >= 2440):
        print("Total loss is: ", loss)
        print("Students withdrawn: ", withdraw)
        print("The number of safe students is 0")
        print("Every one is infected")
        print("100 percent infected")
        continue
    infected_percent = infected * 100 / total
    safe = total - int(infected)
    loss += safe * 0.05 * fee
    print("The loss is: ", loss)
    print("The infected percentage is: ", infected_percent)
    print("Students withdrawn: ", withdraw)
    print("The number of students infected is: ", infected)
    print("Students who are safe: ", safe)
